pkg/util/containers/metrics/cgroup_metrics.go

```diff
@@ -104,6 +104,9 @@ func (c ContainerCgroup) Mem() (*CgroupMemStat, error) {
 			ret.TotalActiveFile = v
 		case "total_unevictable":
 			ret.TotalUnevictable = v
+		case "failcnt":
```

Comment:
I think this is where the agent parses the statsfile and if it sees a value for failcnt will report it

Do we want to be reporting 0s

Reply:
IMO if the metric is not available in the statsfile, we should not report it at all